tests/generic/mongocollection-aggregatecursor-007.phpt

```diff
@@ -38,6 +39,7 @@ for ($i = 0; $i < 103; $i++) {
     $collection->insert(array('article_id' => $i));
 }
 
+// This assumes that the server's batch size defaults to 101 for aggregate
```

Comment:
Do we still want to rely on that, or perhaps also have a test where we force it to 101? The main thing here is that we make sure we test that getMore works.